### PR TITLE
Make tables horizontally scrollable on overflow and add wordbreak in tables

### LINKED ISSUE #4106: Ensure that antd tables are not too wide
Can be provoked by zooming in.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR adds CSS rule to make tables horizontally scrollable when the screen/container of the table is not wide enough to display the whole table. It also adds the word-break property to the table data to force long words in a table cell to be written in multiple lines.

Additionally, this PR adds the possibility of clicking on the dataset name in the dataset table (dashboard) to open the dataset in view mode.

### URL of deployed dev instance (used for testing):
- https://maketablesscrollable.webknossos.xyz
### Steps to test:
- Open the dataset table in the dashboard. 
- Resize the window to have a small width. The dataset table should now be horizontally scrollable. The scrollbar should be at the bottom of the view. Long dataset names should also be displayed in multiple lines. 
- Click on the name of a dataset. It should open up the dataset in view mode.

### Issues:
- fixes #4106 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by MichaelBuessemeyer** [comment]
@philippotto This PR is also be ready for your review :slightly_smiling_face:

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 - Added the possbility to enforce a certain magnification range for tasks (can be configured in the corresponding task type). [#4101](https://github.com/scalableminds/webknossos/pull/4101)
 - Added the possibility for admins to add experience domains while creating new tasks. [#4119](https://github.com/scalableminds/webknossos/pull/4119)
 - Added a histogram to the dataset settings for each layer. It simplifies adjusting the brightness and contrast of a layer and replaces the brightness and contrast slider. [#4105](https://github.com/scalableminds/webknossos/pull/4105)
+- The dataset table in the dashboard is now horizontally scrollable if the window is not wide enough. Additionally, clicking on the name of a dataset in the table opens of the dataset in view mode. [#4136](https://github.com/scalableminds/webknossos/pull/4136)
```
Comment (philippotto) [inline CHANGELOG.md]: I think the `of` in the second sentence has to be removed.

**Comment by MichaelBuessemeyer** [comment]
@philippotto I applied your feedback. As tables are often used in wk, and I did not check whether all tables look fine, with the fix, I only applied it to the dataset table.

If you think there should be a fix like this for all tables please let me know.

P.S.: If the second paragraph does not apply, my PR is ready for your review :)

**Comment by philippotto** [comment]
> @philippotto I applied your feedback. As tables are often used in wk, and I did not check whether all tables look fine, with the fix, I only applied it to the dataset table.

The table itself looks good now, but the horizontal scrollbar is not visible for me :/ Does it appear for you?

**Comment by MichaelBuessemeyer** [comment]
> The table itself looks good now, but the horizontal scrollbar is not visible for me :/ Does it appear for you?

Yes, it works for me. The scrollbar is displayed at the bottom directly above the pagination. It works in firefox and chrome. :thinking:

File: frontend/javascripts/dashboard/advanced_dataset/dataset_table.js [hunk 2]
```diff
@@ -145,6 +146,8 @@ class DatasetTable extends React.PureComponent<Props, State> {
         }
         onChange={this.handleChange}
         locale={{ emptyText: this.renderEmptyText() }}
+        className="test1"
```
Comment (philippotto) [inline frontend/javascripts/dashboard/advanced_dataset/dataset_table.js]: Should probably be removed?

**Comment by philippotto** [comment]
> > The table itself looks good now, but the horizontal scrollbar is not visible for me :/ Does it appear for you?
> 
> Yes, it works for me. The scrollbar is displayed at the bottom directly above the pagination. It works in firefox and chrome.

Ok, I found the scrollbar :D Ideally, it wouldn't be at the very bottom of the page but at the bottom of the window. But unless you feel like massaging the CSS further, I think it's sufficient for now.

File: frontend/javascripts/dashboard/advanced_dataset/dataset_table.js (hunk 2 above)
Comment (MichaelBuessemeyer) [reply frontend/javascripts/dashboard/advanced_dataset/dataset_table.js]: :see_no_evil: :rofl:

**Comment by MichaelBuessemeyer** [comment]
> Ok, I found the scrollbar :D Ideally, it wouldn't be at the very bottom of the page but at the bottom of the window. 

I tried it some time, but could not get it working :/

**Comment by MichaelBuessemeyer** [comment]
@fm3 Thanks for also doing the frontend part :slightly_smiling_face:

**Pull request merged**